src/main/java/dev/lu15/voicechat/network/voice/GroupManager.java

```diff
@@ -0,0 +1,33 @@
+package dev.lu15.voicechat.network.voice;
+
+import dev.lu15.voicechat.network.minecraft.Group;
+import net.minestom.server.entity.Player;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.UUID;
+
+public class GroupManager {
+
+    public HashMap<UUID, Group> groups = new HashMap<>();
+    public HashMap<Player, UUID> playerGroups = new HashMap<>();
+    public HashMap<UUID, ArrayList<Player>> groupPlayers = new HashMap<>();
+    public HashMap<UUID, String> groupPassword = new HashMap<>();
```

Comment:
These should be private and use the `Map` interface instead of the `HashMap` implementation.